ipynbs/Chasing_Platypus_via_MG-RAST.ipynb

```diff
@@ -401,25 +401,22 @@
     "    country_data = '\\n'.join(country_data)\n",
     "    \n",
     "    # this is a JavaScript dictionary of colors ... datamaps has an odd API\n",
-    "    color_data = '\\n'.join([\"'%s': '%s',\" % (a, a) for a in df.Color])\n",
-    "\n",
-    "    html_temp = \"\"\"<html lang=\"en\"> \n",
-    "    <head>\n",
-    "        <script type=\"text/javascript\" src=\"http://d3js.org/d3.v3.min.js\" charset=\"utf-8\"></script>\n",
-    "        <script type=\"text/javascript\" src=\"http://d3js.org/topojson.v1.min.js\"></script>\n",
-    "        <script type=\"text/javascript\" src=\"http://datamaps.github.io/scripts/0.4.0/datamaps.all.js\"></script>\n",
-    "    </head>\n",
-    "\n",
-    "    <body>\n",
-    "        <script type=\"text/javascript\" src=\"http://d3js.org/d3.v3.min.js\" charset=\"utf-8\"></script>\n",
-    "        <script type=\"text/javascript\" src=\"http://d3js.org/topojson.v1.min.js\"></script>\n",
-    "        <script type=\"text/javascript\" src=\"http://datamaps.github.io/scripts/0.4.0/datamaps.all.js\"></script>\n",
-    "        <h1>{{ name }}</h1>\n",
-    "        <div id=\"container\" style=\"position: relative; width: 500px; height: 300px;\"></div>\n",
-    "        <script>\n",
+    "    color_data = '\\n'.join([\"'%s': '%s',\" % (a, a) for a in df.Color.unique()])\n",
+    "\n",
+    "    html_template = \"\"\"\n",
+    "    <h1>Thylacinus</h1>\n",
```

Comment:
Updated, thanks for catching this!

On (Aug-01-15| 7:30), Antonio Gonzalez wrote:
> > -    "        <script type=\"text/javascript\" src=\"http://d3js.org/d3.v3.min.js\" charset=\"utf-8\"></script>\n",
> > -    "        <script type=\"text/javascript\" src=\"http://d3js.org/topojson.v1.min.js\"></script>\n",
> > -    "        <script type=\"text/javascript\" src=\"http://datamaps.github.io/scripts/0.4.0/datamaps.all.js\"></script>\n",
> > -    "    </head>\n",
> > -    "\n",
> > -    "    <body>\n",
> > -    "        <script type=\"text/javascript\" src=\"http://d3js.org/d3.v3.min.js\" charset=\"utf-8\"></script>\n",
> > -    "        <script type=\"text/javascript\" src=\"http://d3js.org/topojson.v1.min.js\"></script>\n",
> > -    "        <script type=\"text/javascript\" src=\"http://datamaps.github.io/scripts/0.4.0/datamaps.all.js\"></script>\n",
> > -    "        <h1>{{ name }}</h1>\n",
> > -    "        <div id=\"container\" style=\"position: relative; width: 500px; height: 300px;\"></div>\n",
> > -    "        <script>\n",
> > +    "    color_data = '\\n'.join([\"'%s': '%s',\" % (a, a) for a in df.Color.unique()])\n",
> > +    "\n",
> > +    "    html_template = \"\"\"\n",
> > +    "    <h1>Thylacinus</h1>\n",
> 
> All headers say Thylacinus, could you change accordingly?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/Platypus-Conquistador/pull/38/files#r36034058